### PR TITLE
teachers section

### PR DESCRIPTION
**Pull request by Ruslan-Dolibskyi**
need help to fix tablet size teachers-list 2x2

### CODE REVIEW

File: src/partials/teachers-section.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<section class="teachers" id="teachers">
+    <div class="container">
+        <h2 class="section-title">Meet our teachers</h2>
```
Comment (kazamov) [inline src/partials/teachers-section.html:3]: Wrap word teachers into span and add class "teachers"

File: index.html [hunk 2]
```diff
@@ -33,6 +33,8 @@
       <!-- Hero Section -->
       <load src="./src/partials/hero-section.html" />
       <load src="./src/partials/application.html" />
+      <!-- Our teachers -->
+      <load src="./src/partials/teachers-section.html" />
```
Comment (kazamov) [inline index.html:37]: Move teachers section above application section

File: src/partials/teachers-section.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        <p class="section-description">Discover our team of dedicated educators, committed to guiding you on your journey to English proficiency with personalized and engaging instruction</p>
+        <ul class="teachers-list">
+            <li class="teachers-item">
+                <div class="teachers-card">
+                <img 
+                class="teachers-img"
+                srcset="/src/images/teachers/maria.webp 1x,
```
Comment (kazamov) [inline src/partials/teachers-section.html:10]: Add `.` before each path, e.g. `./src/images/teachers/maria.webp 1x` and so on. Do it for all paths in the document

File: src/partials/teachers-section.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                class="teachers-img"
+                srcset="/src/images/teachers/volodymyr.webp 1x, 
+                /src/images/teachers/volodymyr@2x.webp 2x, 
+                /src/images/teachers/volodymyr@4x.webp 4x"
+                src="/src/images/teachers/volodymyr.webp" 
+                alt="teacher-Volodymyr"
+                width="150"
+                height="150">
+                <h3 class="teachers-name">Volodymyr</h3>
+                <p class="teachers-description"    >Our expert in English for specific purposes, Volodymyr tailors his approach to meet individual career and industry-specific language needs</p>
```
Comment (kazamov) [inline src/partials/teachers-section.html:63]: Remove space after class="teachers-description"

**Review by kazamov — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json
[the remark this answers is not in the document]
Comment (Ruslan-Dolibskyi) [reply .vscode/extensions.json:1]: done

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (Ruslan-Dolibskyi) [reply .vscode/settings.json:1]: done

File: src/partials/teachers-section.html (hunk 1 above)
Comment (Ruslan-Dolibskyi) [reply src/partials/teachers-section.html:3]: done

File: index.html (hunk 2 above)
Comment (Ruslan-Dolibskyi) [reply index.html:37]: done

File: src/partials/teachers-section.html (hunk 3 above)
Comment (Ruslan-Dolibskyi) [reply src/partials/teachers-section.html:10]: done

File: src/partials/teachers-section.html (hunk 4 above)
Comment (Ruslan-Dolibskyi) [reply src/partials/teachers-section.html:63]: done

**Comment by Ruslan-Dolibskyi** [comment]
I fix all your comments by section

**Review by kazamov — APPROVED** [review]

**Pull request merged**